config/ompi_check_ucx.m4

```diff
@@ -41,16 +45,27 @@ AC_DEFUN([OMPI_CHECK_UCX],[
                                                     [ompi_check_ucx_dir=])],
                                              [true])])
                   ompi_check_ucx_happy="no"
+                  ompi_check_ucg_happy="no"
+                  ucx_libs="-luct -lucm -lucs"
                   AS_IF([test -z "$ompi_check_ucx_dir"],
                         [OPAL_CHECK_PACKAGE([ompi_check_ucx],
                                    [ucp/api/ucp.h],
                                    [ucp],
                                    [ucp_cleanup],
-                                   [-luct -lucm -lucs],
+                                   [$ucx_libs],
                                    [],
                                    [],
                                    [ompi_check_ucx_happy="yes"],
                                    [ompi_check_ucx_happy="no"])
+                         OPAL_CHECK_PACKAGE([ompi_check_ucg],
```

Comment:
It would probably be good to put some comments here explaining the scheme here: UCX vs. UCG, why you have OPAL_CHECK_PACKAGE twice, ...etc.